openmdao/components/multifi_meta_model.py

```diff
@@ -118,7 +124,10 @@ def add_input(self, name, val=1.0, **kwargs):
             For advanced users only. ID or color for this variable, relevant for
             reconfigurability. Default is 0.
         """
-        metadata = super(MultiFiMetaModelUnStructured, self).add_input(name, val, **kwargs)
+        parent = MultiFiMetaModelUnStructured
```

Comment:
'parent' isn't the best name to use here.  MultiFiMetaModelUnStructured is the class of self, and super(..., self) gives us the parent class.

Reply:
wait, what? oh, did Ken maybe change this? what should it say?

Reply:
yeah, just change it to 'name' or 'item' or something

Reply:
I went with "item." Fixed.